rotkehlchen/tests/api/test_location_asset_mappings.py

```diff
@@ -196,7 +198,7 @@ def test_location_asset_mappings_pagination(rotkehlchen_api_server: 'APIServer')
     )
     result = assert_proper_response_with_result(response=response)
     assert result['entries'] == [all_mappings[1]]
-    assert result['entries_found'] == 1500
+    assert result['entries_found'] == NUM_ASSETS_MAPPINS_V1_32
```

Comment:
```suggestion
    assert result['entries_found'] == NUM_ASSETS_MAPPINGS_V1_32
```

same in all the places where you use it